package.json

```diff
@@ -38,7 +39,6 @@
     "@octokit/rest": "19.0.3",
     "@sentry/nextjs": "7.8.1",
     "downshift": "6.1.7",
-    "focus-visible": "5.2.0",
```

Comment:
Taking advantage of this PR to remove the `focus-visible` dependency since [Chakra UI already ensures components show focus outline only when interacting with the keyboard since v2.2.0](https://chakra-ui.com/changelog#06-06-2022).